docs/quick-start-react-native.md

```diff
@@ -19,12 +19,24 @@ npm i --save-dev reactotron-react-native
 I like a separate file for initializing.  Create `ReactotronConfig.js` in your editor of choice and paste this:
 
 ```js
-import Reactotron from 'reactotron-react-native'
-
-Reactotron
-  .configure() // controls connection & communication settings
-  .useReactNative() // add all built-in react native plugins
-  .connect() // let's connect!
+import Reactotron, {
+  trackGlobalErrors,
+  openInEditor,
+  overlay,
+  asyncStorage,
+  networking
+} from 'reactotron-react-native'
+
+  Reactotron
+    .configure({
+      name: 'React Native Demo'
+    })
+  .use(trackGlobalErrors())
+  .use(openInEditor())
+  .use(overlay())
+  .use(asyncStorage())
+  .use(networking())
```

Comment:
Why are you suggesting to do them individually vs the `.useReactNative()` way? It might be worth calling out that you can do them individually but I think `.useReactNative()` makes it easier.

Reply:
I 100% agree, but in my experience it isn't uncommon that `.useReactNative()` throws saying that it is not a function. Though when using these individually everything checks out. After reading through some of the repo issues it appears this solution has been suggested to help users avoid this error. For what it's worth, the most recent project I've used this with is using v1.12.2.